client/client.go

```diff
@@ -1110,21 +1119,31 @@ func (c *Client) run() {
 	}
 }
 
+// nodeMapHashes returns the hashes of the passed Node's attribute and metadata
+// maps.
+func nodeMapHashes(node *structs.Node) (attrHash, metaHash uint64, err error) {
```

Comment:
Do you need named variables in the return? You re-initialize them below, or just return 0.

Reply:
I want to be explicit on which is which hash because it is important to pass them in to the `watchNodeUpdates` call correctly